integrations/heap/lib/index.js

```diff
@@ -61,7 +62,7 @@ Heap.prototype.initialize = function() {
     });
   };
 
-  window.heap.load(this.options.appId);
+  window.heap.load(this.options.appId, this.options);
```

Comment:
I wasn't sure whether it was best to just pass the `options` dictionary straight through like this (making it easier to support future options), or whether it's standard practice to explicitly list the options, something like:

```javascript
window.heap.load(this.options.appId,
  { secureCookie: this.options.secureCookie }
);
```